packages/contextual-help/README.md

```diff
@@ -74,6 +74,20 @@ By default a `sp-contextual-help` will render its popover at the `bottom-start`
 </sp-contextual-help>
 ```
 
+## Custom max width
+
+Contextual help allows for a custom maximum width to be set using the `--mod-spectrum-contextual-help-popover-maximum-width` custom property.
```

Comment:
Need to mention that max-width can be always greater or equal to 100 and not less!